### PR TITLE
refactor(SimpleInfoItem): Migrate to TypeScript

### LINKED ISSUE #204: Convert SimpleInfoItem from JavaScript to TypeScript

### PR DESCRIPTION
**Pull request by ofredstie**
Migrate `SimpleInfoItem` from JavaScript to TypeScript.

Closes #204

### CODE REVIEW

File: components/common/atoms/SimpleInfoItem.tsx [hunk 1]
```diff
@@ -1,9 +1,19 @@
 import PropTypes from "prop-types";
-import React from "react";
+import React, { ReactElement } from "react";
```
Comment (dansvend) [inline components/common/atoms/SimpleInfoItem.tsx:2]: This line can be removed.

File: components/common/atoms/SimpleInfoItem.tsx [hunk 2]
```diff
@@ -18,7 +28,9 @@ const styles = StyleSheet.create({
   },
 });
 
-const SimpleInfoItem = ({ item }) => (
+const SimpleInfoItem: React.FC<SimpleInfoItemProps> = ({
```
Comment (dansvend) [inline components/common/atoms/SimpleInfoItem.tsx:31]: Let's remove `React.FC`.

File: components/common/atoms/SimpleInfoItem.tsx [hunk 3]
```diff
@@ -18,7 +28,9 @@ const styles = StyleSheet.create({
...
+  item,
+}): ReactElement => (
```
Comment (dansvend) [inline components/common/atoms/SimpleInfoItem.tsx:33]: Let's remove the return type.

File: components/common/atoms/SimpleInfoItem.tsx [hunk 4]
```diff
@@ -1,9 +1,19 @@
 import PropTypes from "prop-types";
-import React from "react";
+import React, { ReactElement } from "react";
 import { StyleSheet, Text, View } from "react-native";
 
 import * as Colors from "../../../constants/colors";
 
+interface Item {
+  key: string;
```
Comment (dansvend) [inline components/common/atoms/SimpleInfoItem.tsx:8]: I see the `Item` type require a `key` prop but it's not used anywhere.

File: components/common/atoms/SimpleInfoItem.tsx [hunk 5]
```diff
@@ -1,9 +1,19 @@
 import PropTypes from "prop-types";
-import React from "react";
+import React, { ReactElement } from "react";
 import { StyleSheet, Text, View } from "react-native";
 
 import * as Colors from "../../../constants/colors";
 
+interface Item {
```
Comment (dansvend) [inline components/common/atoms/SimpleInfoItem.tsx:7]: This line can be removed.

**Review by dansvend — CHANGES_REQUESTED** [review]

**Comment by ofredstie** [comment]
These changes are implemented as part of #209

**Pull request closed without merging**